e2e/components/cms-widget/cms-widget.po.js

```diff
@@ -0,0 +1,95 @@
+const elements = {
+    cmsWidget: '#widget-toggle',
+    chplPublicUserGuide: '=CHPL Public User Guide',
+    cmsIdReverseLookup: '=CMS ID Reverse Lookup',
+    progressBar: '.progress-bar',
+    certId: '#get-ehr-cert-id',
+    removeProducts: '#cms-remove-all',
+    baseCriteria: '=Base Criteria',
+    compareProducts: '#cms-compare',
+    cmsCertificationId: 'strong.ng-binding',
+    compareWidgetDropdown: '#compare-widget-dropdown',
+    missingBaseCriteriaListOr: '.cms-widget__missing-or',
+    missingBaseCriteriaListAnd: '.cms-widget__missing-and',
+    downloadPDF: '#download-ehr-cert',
+    processingSpinner: '.fa.fa-spinner.fa-spin',
+};
+
+const asserts = {
+    noProductsSelected: '//*[@id="widget-dropdown"]/ai-cms-widget-display/div/div[1]',
+};
+
+class CmsWidgetComponent {
+    constructor () { }
+
+    get cmsWidget () {
+        return $(elements.cmsWidget);
+    }
+
+    get compareProductsButton () {
+        return $(elements.compareProducts);
+    }
+
+    get chplPublicUserGuideLink () {
+        return $(elements.chplPublicUserGuide);
+    }
+
+    get cmsIdReverseLookupLink () {
+        return $(elements.cmsIdReverseLookup);
+    }
+
+    get progressBar () {
+        return $(elements.progressBar);
+    }
+
+    get getCertIdButton () {
+        return $(elements.certId);
+    }
+
+    get removeProductsButton () {
+        return $(elements.removeProducts);
+    }
+
+    get baseCriteriaLink () {
+        return $(elements.baseCriteria);
+    }
+
+    get noProductsSelectedText () {
+        return $(asserts.noProductsSelected);
+    }
+
+    get cmsCertificationIdText () {
+        return $(elements.cmsCertificationId);
+    }
+
+    get compareWidgetDropdown () {
+        return $(elements.compareWidgetDropdown);
+    }
+
+    get missingBaseCriteriaListOr () {
+        return $(elements.missingBaseCriteriaListOr);
+    }
+
+    get missingBaseCriteriaListAnd () {
+        return $(elements.missingBaseCriteriaListAnd);
+    }
+
+    get downloadPDFButton () {
```

Comment:
PDF isn't an acronym, or if it is, it's not used that way
```suggestion
    get downloadPdfButton () {
```

Reply:
Updated